tests/testthat/test-multitables.R

```diff
@@ -653,22 +364,6 @@ with_test_authentication({
         })
     })
 
-    test_that("We can get an xlsx tab book", {
-        skip_on_local_backend("Vagrant host doesn't serve files correctly")
-        f <- tempfile()
-        out <- tabBook(mult, data = ds, output_format = "xlsx", file = f)
-        expect_true(file.exists(out))
-    })
-
-    test_that("We can get an json tab book", {
-        skip("multitables and multiple response need more work.")
-        skip_on_local_backend("Vagrant host doesn't serve files correctly")
-        book <- tabBook(mult, data = ds)
-        expect_is(book, "TabBookResult")
-        expect_identical(dim(book), c(ncol(ds), 3L))
-        expect_identical(names(book), names(variables(ds)))
-    })
-
     test_that("team-sharing of multitables", {
```

Comment:
It looks like a bunch of tests were removed here?

Reply:
Just moved to a file dedicated to `tabBook`